src/main/java/co/cask/hydrator/python/transform/JythonPythonExecutor.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright © 2019
+ * Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.hydrator.python.transform;
+
+import co.cask.cdap.api.data.format.StructuredRecord;
+import co.cask.cdap.etl.api.Emitter;
+import co.cask.hydrator.common.script.ScriptContext;
+import org.python.core.Py;
+import org.python.core.PyCode;
+import org.python.core.PyException;
+import org.python.util.PythonInterpreter;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.Map;
+
+/**
+ * Executes python code using Jython library
```

Comment:
why do we need to keep both libraries?

Reply:
This was required by Albert since Jython will not require any installation. 

_Interpreted mode_:
Python code is executed via jvm, hence C based libs (e.g. numpy) and Python3 syntax are not supported.

_Native mode:_
Python process is started, which removes limitations. However, user is required to install py4j library on all hosts.
For Python 2 run ``pip install py4j`` to install it. For Python 3 run ``pip3 install py4j``.